src/enums/orgPersonTypes.ts

```diff
@@ -0,0 +1,4 @@
+export enum OrgPersonTypes {
+  PARTIES = 'parties',
+  DIRECTORS = 'directors'
+}
```

Comment:
What are these used for? Can PartyType or RoleType be used instead?

Reply:
This is used for requests to the API, where we have different end points to retrieve directors vs parties etc.
PartyType is person or org, RoleType is Incorporator, completingParty proprietor etc..
I can update the name of the enum to `apiOrgPersonType` ?

Reply:
You could rename it, or you could just add a JS Doc comment explaining what this is used for.

Reply:
Updated